### ISSUE TITLE
tldr should not return 0 when a TLDR page is not found

### ISSUE THREAD

**Issue by letompouce** [tldr-pages/tldr#4227]
## Description

When `tldr` doesn't find a TLDR page, it should not return 0.

## Steps to reproduce

Feed `tldr` with a trainspotting-powered argument, observe its return code.

## Expected result

`tldr` does not return 0

## Actual restult

`tldr` return 0 whatever you feed it with.

## Example 

```
$ tldr Choose a life. Choose a job. Choose a career. Choose a family.  2> /dev/null && echo true          
true
```

## Why would this been addressed

For instance, when a page is not found in your [$LANG language](https://github.com/tldr-pages/tldr/blob/master/CLIENT-SPECIFICATION.md#language), this would allow to fallback to `en` (which supposedly has more references, being [the default language](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#default-language-for-newly-added-pages)) , such as:

```shell
tldr ss || tldr --language en ss
```
### Actual result

```
$ tldr ss || tldr --language en ss
`ss` documentation is not available. Consider contributing Pull Request to https://github.com/tldr-pages/tldr
```
### Expected result

```
$ tldr ss || tldr --language en ss

  ss

  Utility to investigate sockets.
[...]
```

**Comment by sbrl** [comment]
Hey there, @letompouce! Thanks for the report. Indeed, that does sound like a bug - I agree the exit code should reflect whether it finds a page or not. This makes it easier to drive via a script - and indeed [my tldr-missing-pages script](https://gitlab.com/sbrl/bin/raw/master/tldr-missing-pages) relies on it.

However, this is a client-specific feature, and this repository is just for the pages themselves. Do you know which client you're using (or perhaps how you installed it)? I can direct you to the correct repository to report this if you are unsure :-)

**Comment by samuel-w** [comment]
Judging by the syntax of the error message, the client is tldr-python-client, which is fixed by tldr-pages/tldr-python-client#129

**Comment by sbrl** [comment]
Thanks, @samuel-w :heart:

We've also got #4246 open against the spec to improve that too. Once that PR is merged, we can close this issue I think.

**Comment by letompouce** [comment]
Hi,

Sorry for the delay, summertime and brain meltdown!

That was both: Haskell client from the [Debian Buster package](https://packages.debian.org/buster/tldr) (`0.4.0-2+b2` at the time) ; and the python client (`1.0.0` from pip).

Thanks you all for the issue forwarding, python client fix, specification update!
